Stratis.Bitcoin.Tests/Utilities/NodeTcpListenerStub.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Stratis.Bitcoin.Tests.Utilities
 {
-    internal class NodeTcpListenerStub : IDisposable
+    public class NodeTcpListenerStub : IDisposable
```

Comment:
Well this is a test class but same comment can apply to use [assembly: InternalsVisibleTo("Stratis.Bitcoin.Tests")]

Reply:
Added `[assembly: InternalsVisibleTo("Stratis.Bitcoin.Features.Wallet.Tests")]`. Not sure about this one though as preferable these files should carry no knowledge of the features out there.

Reply:
Yeah you are right actually, and the tcp stub is something we do want to be public